client/components/theme/index.jsx

```diff
@@ -172,8 +176,12 @@ const Theme = React.createClass( {
 								context: 'singular noun, the currently active theme'
 							} ) }</span>
 						}
-						{ price && ! purchased &&
-							<span className="theme-badge__price">{ price }</span>
+						{
+							config.isEnabled( 'jetpack/pijp' )
+							? ! this.props.isJetpack && price && ! purchased &&
```

Comment:
Thanks for this discussion, copy and small details in a critical path can have a big impact. 

For reference, here's the current state of this PR:

<img width="189" alt="screen shot 2017-06-19 at 11 24 31" src="https://user-images.githubusercontent.com/4389/27278514-e65581a0-54e1-11e7-99c0-5a6dce3caea7.png">



Also, here's a Premium Theme call to action button  (both logged out and logged in):

<img width="192" alt="screen shot 2017-06-19 at 11 17 07" src="https://user-images.githubusercontent.com/4389/27278203-ea723130-54e0-11e7-8f99-08ff5ce8bc89.png">

Notice that even if the theme is paid, the action is worded to be generic, having only the price as an indicator. The main text provides the action. We chose this wording after months of reviews and tests, so I'd like to preserve it unless we can do an A/B test uniquely on the wording. 

My advice at this stage would be to do this:

<img width="205" alt="screen shot 2017-06-19 at 11 20 11" src="https://user-images.githubusercontent.com/4389/27278329-4e27af34-54e1-11e7-86d5-56107e9e1786.png">

_(Note that the font size has been reduced to `12px`)_

Which has the benefit of keeping the wording clear and consistent, but also using an indicator for the "Plan" toggle where the price is meant to be (again, consistency). I understand it's not "crystal clear", but I think it's more important for now to keep the wording and the clarity of the action.